web/packages/teleterm/src/ui/ConnectMyComputer/connectMyComputerContext.tsx

```diff
@@ -116,8 +119,8 @@ export const ConnectMyComputerContextProvider: FC<{
         rootCluster,
         configService,
         mainProcessClient.getRuntimeSettings()
-      ),
-    [configService, mainProcessClient, rootCluster]
+      ) || isAgentConfigured,
```

Comment:
should `isAgentConfigured` be the first to be checked so we can short circuit faster?

Reply:
Actually, it won't change much since `canUseConnectMyComputer` is a very cheap function and we have this in `useMemo`, so I would keep it as is.